otx/cli/tools/train.py

```diff
@@ -101,6 +91,7 @@ def parse_args():
     parser.add_argument(
         "--save-model-to",
         required=False,
+        default="results",
```

Comment:
Maybe "output_model" or "trained_model"

Reply:
Since this folder also includes configurations (i.e.`template.yaml`, `model.py`), I named this name as `results`